### ISSUE TITLE
Move Cargo details to workspace

### ISSUE THREAD

**Issue by addisoncrump** [AFLplusplus/LibAFL#1304]
Right now, certain details of crates in the repository are duplicating Cargo keys. This includes things like dependencies, edition, documentation links, readmes, etc.

We should move these items to the root of the workspace.

**Comment by npranav7619** [comment]
Can you provide more details

**Comment by addisoncrump** [comment]
Yes -- we want to move common Cargo.toml fields (like `authors`, `license`, `edition`, etc.) to the workspace Cargo.toml, then use `authors.workspace`, etc. in the Cargo.toml files.

**Comment by gerceboss** [comment]
@addisoncrump ,can you please assign me this issue?

**Comment by addisoncrump** [comment]
We don't really "assign" an issue; please just open a pull request with progress.
